jenkinsfiles/Windows_Check.groovy

```diff
@@ -164,8 +176,15 @@ pipeline {
                                     githubStatus.setPending(this, "Jenkins/Win_Check")
                                     deleteDir()
 
-                                    checkout changelog: false, poll: false, scm: [$class: 'GitSCM', branches: [[name: '${Commit_id}']], doGenerateSubmoduleConfigurations: false, extensions: [[$class: 'RelativeTargetDirectory', relativeTargetDir: 'src'], [$class: 'CloneOption', timeout: 200]], submoduleCfg: [], userRemoteConfigs: [[refspec: "+refs/pull/${PR_number}/head:refs/remotes/origin/PR-${PR_number}", url: 'https://github.com/oneapi-src/oneDPL.git']]]
-
+                                    bat script: """
+                                        d:
+                                        cd ${env.WORKSPACE}
+                                        git clone https://github.com/oneapi-src/oneDPL.git src
+                                        cd src
+                                        git config --local --add remote.origin.fetch +refs/pull/${env.PR_number}/head:refs/remotes/origin/pr/${env.PR_number}
```

Comment:
Hi @zheltovs 
Thank you for the review. I have solved other reviews but this one. Because for this repo, oneapi-src, I see many developers are using forked repo to commit the source code. In this case the commit can not be found unless we fetch the refs for Pull Request.